production/helm/loki/README.md

```diff
@@ -16,6 +16,18 @@ $ helm repo add loki https://grafana.github.io/loki/charts
 $ helm upgrade --install loki loki/loki
 ```
 
+## Deploy Loki only
+
+```bash
+$ helm upgrade --install loki loki/loki --set 'promtail.enabled=false'
```

Comment:
why not separate totally? Then we can deploy `loki/loki` or `loki/promtail`.

Reply:
@daixiang0 You mean to say remove the `requirements.yaml` so users have to deploy both? I can do that if that's what the Loki team wants. It does take an extra install step and would require the user to specify the loki service name in the promtail configuration.

Reply:
You can totally separate them so there is a loki/loki and loki/promtail, and if you wanted to still enable a single aggregate, make a wrapper chart that has no templates of its own, and requires.yaml includes both the other two charts. Similar to https://github.com/helm/charts/tree/master/stable/elastic-stack

Reply:
@davkal @tomwilkie  Preference? Separate with aggregate chart would actually be backwards compatible with current custom values.yaml. It would also allow future separation of loki into ingestor, distributor, and querier.

Recommendations for aggregate chart name? loki-stack, loki-bundle, loki-parent, loki-charts? Or maybe aggregate chart called loki and loki child chart called something else?

Reply:
I'm a bit partial to loki-stack.

Reply:
+1 on separating out `loki/promtail` and aggregate chart `loki-stack`. Good effort!

Reply:
loki-stack chart added